src/common/avatars/BeverageAvatar.js

```diff
@@ -0,0 +1,31 @@
+// @flow
+
+import type { BaseAvatarProps } from './BaseAvatar';
+
+import * as React from 'react';
+import CONFIG from '../../config';
+import BaseAvatar from './BaseAvatar';
+
+type Props = {
```

Comment:
Doesn't this work?
```
type Props = {
  ...BaseAvatarProps,
  beverageId: string,
};
```

Reply:
idk why, but no, it doesn't. it treats props.size as undefined. I still can't get a clear difference between & and ... in flow context, sometimes one works better and vice versa.